Kernel/Arch/x86/Processor.h

```diff
@@ -27,6 +27,12 @@ VALIDATE_IS_X86()
 
 namespace Kernel {
 
+// FIXME: Share this code with aarch64/Processor.h
```

Comment:
Can't we just put this in `Kernel/Arch/Processor.h`?

Reply:
I did that initially, but it caused problems for the x86_64 build, because some drivers pull in `Kernel/Arch/x86/Processor.h` directly... Might as well add a commit where I change the include for those drivers.

Reply:
The code is now shared in `Arch/Processor.cpp`.